themes/default/index.html

```diff
@@ -226,9 +246,17 @@
                 <!-- Body content -->
                 <tr>
                   <td class="content-cell">
-                    <h1><%- greeting %> <%- name %>,</h1>
-                    <p><%- intro %></p>
-                    
+                    <% if (locals.title) { %>
```

Comment:
Nice one, done. Worth noting there are currently slight differences in the greeting end character between themes (`,` in default, `!` in neopolitan) that are wiped out by this change. I think it's better these are all consistent anyway.